gax/src/createApiCall.ts

```diff
@@ -72,9 +76,22 @@ export function createApiCall(
     callOptions?: CallOptions,
     callback?: APICallback
   ) => {
-    const thisSettings = settings.merge(callOptions);
-
     let currentApiCaller = apiCaller;
+
+    let thisSettings: CallSettings;
+    if (currentApiCaller instanceof StreamingApiCaller) {
+      const gaxStreamingRetries =
+        currentApiCaller.descriptor?.gaxStreamingRetries;
+      // If Gax streaming retries are enabled, check settings passed at call time and convert parameters if needed
+      const thisSettingsTemp = checkRetryOptions(
```

Comment:
I'm nitpicking on names, but `thisSettingsTemp` does not help the reader understand what's going on. What will the function return?